contracts/interchain-token-service/src/error.rs

```diff
@@ -21,4 +21,7 @@ pub enum ContractError {
     TokenAlreadyRegistered = 15,
     InvalidTokenMetaData = 16,
     InvalidTokenId = 17,
+    TokenAlreadyDeployed = 18,
+    EmptyTokenName = 19,
```

Comment:
we don't need `EmptyTokenName` and `EmptyTokenSymbol` error here if we use `validate_token_metadata`